code/modules/ghostroles/spawner/human/responseteams/skrell/qukala.dm

```diff
@@ -0,0 +1,48 @@
+/datum/ghostspawner/human/ert/qukala
```

Comment:
Only the leadership role in a species restricted ERT may be whitelisted.
All the other roles must be accessible without a whitelist.